spinalcordtoolbox/scripts/sct_dmri_separate_b0_and_dwi.py

```diff
@@ -200,7 +200,13 @@ def main(args=None):
     # Average b=0 images
     if average:
         printv('\nAverage b=0...', verbose)
-        run_proc(['sct_maths', '-i', b0_name + ext, '-o', b0_mean_name + ext, '-mean', 't'], verbose)
+        img = Image(b0_name + ext)
+        out = img.copy()
+        dim_idx = 3 # 3 dimensions + time
+        if dim_idx + 1 > len(np.shape(img.data)):
+            out.data = out.data[..., np.newaxis]
```

Comment:
Do we need this check here? 

While yes, it was present in `sct_maths`, I think we can guarantee that the `b=0` images should always be of shape [x, y, z, t] because of the line immediately above:

https://github.com/neuropoly/spinalcordtoolbox/blob/6129ec7ccb16b777462d57276e1f63498a1f9ef2/spinalcordtoolbox/scripts/sct_dmri_separate_b0_and_dwi.py#L198


If the condition `if dim_idx + 1 > len(np.shape(img.data)):` ever isn't true, then there's a bigger problem... (so just appending a new axis would be hiding that).